### PR TITLE
Flow:Multiple media entries

### PR DESCRIPTION
**Pull request by AmishaBisht**
## Summary
 Target issue is  #3088 
 
created a single media entry for same url

### CODE REVIEW

File: lib/glific/flows/contact_action.ex [hunk 1]
```diff
@@ -371,23 +375,31 @@ defmodule Glific.Flows.ContactAction do
...
-          organization_id: context.organization_id
-        }
-        |> Messages.create_message_media()
-
-      {type, message_media.id}
+    db_url = from(m in MessageMedia, where: m.url == ^url) |> Repo.all()
+
+    case db_url do
+      [] ->
+        if is_nil(url) do
```
Comment (AkhileshNegi) [inline lib/glific/flows/contact_action.ex]: why `is_nil`  url check is after we are querying the db and not before that

File: lib/glific/flows/contact_action.ex [hunk 2]
```diff
@@ -371,23 +375,31 @@ defmodule Glific.Flows.ContactAction do
...
-          source_url: url,
-          thumbnail: url,
-          flow: :outbound,
-          caption: MessageVarParser.parse(caption, message_vars),
-          organization_id: context.organization_id
-        }
-        |> Messages.create_message_media()
-
-      {type, message_media.id}
+    db_url = from(m in MessageMedia, where: m.url == ^url) |> Repo.all()
```
Comment (AkhileshNegi) [inline lib/glific/flows/contact_action.ex]: why Repo.all and not Repo.one ?

**Review by AkhileshNegi — CHANGES_REQUESTED** [review]

File: lib/glific/flows/contact_action.ex (hunk 2 above)
Comment (AmishaBisht) [reply lib/glific/flows/contact_action.ex]: Repo.one wasn't working throwing some error

Comment (AmishaBisht) [reply lib/glific/flows/contact_action.ex:378]: fixed it

**Review by AkhileshNegi — APPROVED** [review]

**Pull request merged**